chart/kubeapps/README.md

```diff
@@ -71,9 +73,11 @@ To uninstall/delete the `kubeapps` deployment:
 
 ```console
 $ helm delete --purge kubeapps
+$ # Optional: Only if there are no more instances of Kubeapps
+$ kubectl delete crd apprepositories.kubeapps.com
 ```
 
-The command removes all the Kubernetes components associated with the chart and deletes the release.
+The first command removes most of the Kubernetes components associated with the chart and deletes the release. After that, if there are no more instances of Kubeapps in the cluster you can manually delete the `apprepositories.kubeapps.com` CRD used by Kubeapps that is shared for the entire cluster.
```

Comment:
I'd make it more explicit that if you delete the CRD you will break other instances of Kubeapps running in the same cluster.